### PR TITLE
Add support for SIGNALFX_SPAN_TAGS

### PR DESCRIPTION
**Pull request by owais**
Now supports SIGNALFX_SPAN_TAGS env variable which accepts a `;`
separated list of key value pairs. Any such pairs are added to all spans
as tags.

For example,

    export SIGNALFX_SPAN_TAGS='environment=production,version=1.5'

Will add `environment=production` and `version=1.5` tags to every single span.

### CODE REVIEW

**Comment by pjanotti** [comment]
I'm not sure of conventions but SFx.NET uses `SIGNALFX_TRACE_GLOBAL_TAGS` - that said I like the shorter one better. Perhaps we should choose one and make it a standard.

**Comment by owais** [comment]
Oh, I did not know that. Go and Node both use `SIGNALFX_SPAN_TAGS` so probably should go with that :)

File: lib/signalfx/tracing.rb [hunk 1]
```diff
@@ -44,6 +49,17 @@ def configure(tracer: nil,
           Compat.apply
         end
 
+        def process_span_tags(span_tags: {})
+          span_tags_string = ENV['SIGNALFX_SPAN_TAGS']
+          if not span_tags_string.nil?
+            span_tags_string.split(',').each do |t|
```
Comment (pjanotti) [inline lib/signalfx/tracing.rb:55]: fyi SFx.NET current implementation is comma-delimited for the key-value pairs and the pairs themselves are separated by a colon.

Comment (owais) [reply lib/signalfx/tracing.rb:55]: Updated.

**Review by jtmal-signalfx — APPROVED** [review]

**Pull request merged**